app/scripts/directives/nav.js

```diff
@@ -273,10 +294,54 @@ angular.module('openshiftConsole')
         };
 
         $scope.catalogLandingPageEnabled = !Constants.DISABLE_SERVICE_CATALOG_LANDING_PAGE;
-        var select = $elem.find('.selectpicker');
-        var options = [];
 
-        var updateOptions = function() {
+        var contextSelector = $elem.find('.contextselector');
+        var previousContextSelectorValue;
+        contextSelector.append($('<option value="1">Service Catalog</option>'));
```

Comment:
Better to give the values meaningful identifiers like "catalog", "application-console", "cluster-console"

Reply:
If these values are always the same, can they be in the markup instead of adding them in JS?

We had to do this for projects since the options are dynamic.